pkgs/development/compilers/llvm/common/lldb.nix

```
@@ -23,7 +23,8 @@
 , monorepoSrc ? null
 , patches ? [ ]
 , enableManpages ? false
-}:
+, ...
+}@pkgs:
```

```suggestion
}:
```

Not needed any more.